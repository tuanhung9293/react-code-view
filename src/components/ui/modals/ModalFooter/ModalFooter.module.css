@value colors: 'styles/colors.css';
@value separator from colors;

.footer{
  box-sizing: border-box;
  display: flex;
  flex-wrap: wrap;
  justify-content: space-between;
  width: 100%;
  padding: 20px;
}

@media(max-width: 683px){
  .footer {
    padding: 10px 20px;
  }
  .footer > * {
    width: 100%;
    margin: 10px 0;
  }
}