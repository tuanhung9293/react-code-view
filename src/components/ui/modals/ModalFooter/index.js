import React from 'react'
import styles from './ModalFooter.module.css'

function ModalFooter({children}) {
  return (
    <footer className={styles.footer}>
      {children}
    </footer>
  )
}

export default ModalFooter