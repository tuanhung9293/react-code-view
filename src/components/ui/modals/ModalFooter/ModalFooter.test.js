import React from 'react'
import {shallow} from 'enzyme'
import ModalFooter from './'

it('renders children', () => {
  expect(shallow(
    <ModalFooter>
      <p>I feel like a child</p>
    </ModalFooter>
  )).toMatchSnapshot()
})