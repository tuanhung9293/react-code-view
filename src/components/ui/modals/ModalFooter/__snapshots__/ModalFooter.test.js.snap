// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders children 1`] = `
<footer
  className="footer"
>
  <p>
    I feel like a child
  </p>
</footer>
`;
